test/column/editor.test.tsx

```diff
@@ -219,30 +219,53 @@ describe('Editor', () => {
     });
   });
 
-  it.skip('should not steal focus back to the cell after being closed by clicking outside the grid', async () => {
-    const column: Column<unknown> = {
-      key: 'col',
-      name: 'Column',
-      editor() {
-        return <input value="123" readOnly autoFocus />;
+  describe('editor focus', () => {
+    it('should not steal focus back to the cell if the editor is not in the viewport and another cell is clicked', () => {
```

Comment:
Does the test properly fail without the changes?

Reply:
yeah, check the first commit 
https://github.com/adazzle/react-data-grid/runs/3702020217